src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/commands.py

```diff
@@ -18,26 +19,44 @@ def load_command_table(self, _):
         operations_tmpl='azure.mgmt.redis.operations.patch_schedules_operations#PatchSchedulesOperations.{}',
         client_factory=cf_patch_schedules)
 
+    redis_firewall_rules = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.firewall_rules_operations#FirewallRulesOperations.{}',
+        client_factory=cf_firewall_rule)
+
+    redis_linked_server = CliCommandType(
+        operations_tmpl='azure.mgmt.redis.operations.linked_server_operations#LinkedServerOperations.{}',
+        client_factory=cf_linked_server)
+
     with self.command_group('redis', redis_sdk) as g:
         g.custom_command('create', 'cli_redis_create', client_factory=cf_redis,
                          exception_handler=wrong_vmsize_argument_exception_handler)
-        g.command('delete', 'delete')
+        g.command('delete', 'delete', confirmation=True)
         g.custom_command('export', 'cli_redis_export')
         g.command('force-reboot', 'force_reboot')
-        g.custom_command('import-method', 'cli_redis_import_method',
-                         deprecate_info=g.deprecate(redirect='redis import', hide='2.0.34'))
-        g.custom_command('import', 'cli_redis_import_method')
-        g.custom_command('list', 'cli_redis_list')
-        g.command('list-all', 'list', deprecate_info=g.deprecate(redirect='redis list', hide='2.0.34'))
+        g.custom_command('import-method', 'cli_redis_import_method')
```

Comment:
Why did you remove the deprecation. I thought this is a deprecated command? If you deprecate a command, the help text will show its deprecated. The command authoring guidelines go into more detail about deprecation.

Reply:
So deprecated commands are expected to stay in the code forever? I removed the deprecation since the deprecation message was added in the last release, and was getting rid of the entry this time. If deprecated commands are expected to stay forever in the code, i'll add it back

Reply:
I am talking about `redis import-method` not `redis list-all`. It is fine to remove the list-all command completely.

You removed import-methods' deprecation, but added custom deprecation logic. If you want to remove the command completely, remove it, if not leave the logic that deprecates `redis import-method` for `redis import`.